Sources/Files.swift

```diff
@@ -485,6 +485,21 @@ public final class Folder: FileSystem.Item, FileSystemIterable {
     public var subfolders: FileSystemSequence<Folder> {
         return makeSubfolderSequence()
     }
+
+    /// A reference to the folder that is the current working directory
+    public static var current: Folder {
+        return FileSystem(using: .default).currentFolder
```

Comment:
Ah, sorry, shouldn't do code reviews when I'm this tired! 😅 You are completely right 👍